components/org.wso2.carbon.appmgt.hostobjects/src/main/java/org/wso2/carbon/appmgt/hostobjects/APIStoreHostObject.java

```diff
@@ -2505,6 +2505,28 @@ public static boolean jsFunction_isSelfSignupEnabledForTenant(
 
     }
 
+    /**
```

Comment:
Formatting and Java doc comments issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files